src/test/ui/issues/issue-77218.stderr

```diff
@@ -0,0 +1,9 @@
+error[E0308]: mismatched types
+  --> $DIR/issue-77218.rs:3:11
+   |
+LL |     while Some(0) = value.get(0) {
+   |           ^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `()`
```

Comment:
Shouldn't this suggest to be `while let`?

Reply:
Fixed.